modules/dashboards/pages/application-insights.adoc

```diff
@@ -0,0 +1,88 @@
+= Application Insights
+:description: Detailed overview of the Application Insights dashboard for process performance analysis and improvement.
+
+This documentation provides a detailed overview of the Application Insights dashboard, helping stakeholders understand and utilize the insights effectively to improve process performance and efficiency in the BPM context.
```

Comment:
This dashboard

Reply:
no I talk about "this documentation" describing "this dashboard"